emscripten/ARToolKitJS.cpp

```diff
@@ -48,7 +50,7 @@ struct arController {
 	int patt_id = 0; // Running pattern marker id
 
 	ARdouble cameraLens[16];
-
+	AR_PIXEL_FORMAT pixFormat = arVideoGetPixelFormat();
```

Comment:
see this comment https://github.com/kalwalt/jsartoolkit5/pull/1#issuecomment-496445175